### PR TITLE
⬆️ Reinstate Big-DFT

### PR DESCRIPTION
**Pull request by chrisjsewell**
This is in essence a rebase of #128

### CODE REVIEW

**Comment by ltalirz** [comment]
Seems to work (?)

**Comment by chrisjsewell** [comment]
Well not sure, it might be hanging at the install stage: https://github.com/marvel-nccr/quantum-mobile/runs/1260562158?check_suite_focus=true

started a local build to test there as well

**Comment by chrisjsewell** [comment]
Yep there is something broken here: https://github.com/marvel-nccr/quantum-mobile/runs/1260562158

This stalls at `TASK [marvel-nccr.bigdft : Make bigdft executables]`, and I also tried the build locally and got the same result 🤷

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -65,9 +65,7 @@ jobs:
 
     - name: Clean build files on VM
       # only roles that include "when: clean is defined and clean"
-      # TODO wannier90 has a clean task, but it appears to running against the wrong python executable
-      # since it can't find the configparser module, pip installed in earlier task
-      run: tox -e ansible -- --tags quantum_espresso,qm_customizations,simulationbase,ubuntu_desktop --extra-vars "clean=true"
+      run: tox -e ansible -- --tags quantum_espresso,qm_customizations,simulationbase,ubuntu_desktop,wannier90 --extra-vars "clean=true"
```
Comment (chrisjsewell) [inline .github/workflows/build.yml:68]:
```suggestion
      run: tox -e ansible -- --tags quantum_espresso,qm_customizations,simulationbase,ubuntu_desktop --extra-vars "clean=true"
```

**Comment by chrisjsewell** [comment]
Yay, a parsing build with bigdft: https://github.com/marvel-nccr/quantum-mobile/runs/1286287172?check_suite_focus=true

**Pull request merged**